values.yaml

```diff
@@ -0,0 +1,44 @@
+# Default values for infinispan-helm-charts.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image: quay.io/infinispan/server:12.1
+initContainerImage: busybox
```

Comment:
I think we should be using [ubi8-micro](https://catalog.redhat.com/software/containers/ubi8-micro/601a84aadd19c7786c47c8ea)  now.

```
registry.access.redhat.com/ubi8-micro
```

Reply:
Is it public repo? I'm not sure that we can start to bind to the Red Hat registry in the upstream. We can try to find alternative ubi8-micro on the quay.io.

Reply:
Yep it's a public repo. We use images from this repo when creating the [upstream server images](https://github.com/infinispan/infinispan-images/blob/master/server-openjdk.yaml#L17)

Reply:
Thanks, will change.